metabase/metabase
# Search Milestone 2: Add `last_edited_by` search filter

`last_edited_by` filter for Search Milestone 2: #27982 

### Description

Adds `last_edited_by` filter to the search sidebar. The filter reuses the logic for `created_by` with a different key for the search API.


### How to verify

Go to the search sidebar, and try to filter by `Last edited by`. You should only be able to see assets that were last edited by the user you selected. For example, create a Question, then filter the search by `Last edited by: <your name>` and you should see that question.

After filtering, try to refresh the page or paste the URL into a new window to test that hydrating the search filters still works when entering the search directly from a URL. The results should remain the same and should behave in the same way.

### Tests
